Source/Internal/API/APIEndpoints.swift

```diff
@@ -27,12 +27,9 @@ import Foundation
 /// This enum represents all available Virtusize endpoints
 internal enum APIEndpoints {
     case productDataCheck(externalId: String)
-    case productMetaDataHints(
-        externalId: String,
-        imageUrl: URL,
-        storeId: Int?)
+    case productMetaDataHints
     case events
-    case fitIllustrator(storeId: Int, productId: Int)
+    case fitIllustrator(storeId: Int, productId: Int, randomNumber: UInt32? = nil)
```

Comment:
Why are you passing the `randomNumber` here, is it to prevent caching?

Reply:
In order to pass a fixed value for the test function `testFitIllustratorEndpoint_returnExpectedComponents` in `APIEndpointsTests.swift`

My thought was to test if it will return expected URL components for Fit Illustrator, but the value for `_` is random. I'm not exactly sure how to write the assertion for it and that's what I came up with, assigning a fixed value to `_`

Actually I thought of another way to test it. Maybe I could just verify if the random value for `_` is between 0 and 1519982555.
Well, it's also possible that I didn't need to test this part and it's over-tested..

What do you think?

Reply:
So this is only to get rid of any cache. TBH having a `No-Cache` header would be better on the server's end.

So yes I think that testing that parameter between that acceptable change is better than passing extra arguments that don't make sense in the ABI of the app.

Reply:
ok, I removed randomNumber and changed to test that parameter whether it's within the acceptable range.